packages/circuit-ui/components/NotificationBanner/NotificationBanner.tsx

```diff
@@ -254,11 +262,7 @@ export function NotificationBanner({
         <ResponsiveButton {...action} />
       </Content>
       {image && image.src && (
-        <StyledImage
-          alt={image.alt}
-          src={image.src}
-          image={image}
-        ></StyledImage>
+        <StyledImage alt={image.alt} src={image.src} image={image} />
```

Comment:
Should we also spread the rest of the `ImageProps` since we accept them? Otherwise we might wanna declare a more restrictive interface to make sure there's no confusion as to what gets passed to the image

Reply:
We could! Would be nice to support `srcset`, `sizes` and such.

Reply:
Done in 2f95248.